Cargo.toml

```diff
@@ -15,8 +15,8 @@ vendored = ["native-tls/vendored"]
 
 [dependencies]
 bytes = "1"
-native-tls = "0.2.1"
-hyper = { version = "0.14.2", default-features = false, features = ["tcp", "client"] }
+native-tls = { version = "0.2.7", features = ["alpn"] }
```

Comment:
Even if its enabled by default, ALPN support should be behind a feature, since old versions of the TLS backends don't support it.

Reply:
Say more. You mean if built on a system with an old openssl/schannel/security-framework, it will probably fail with link errors of missing symbols?

Reply:
Yes

Reply:
I'm curious how likely it is that installations would be too old. I'd really wish to make it default enabled in reqwest, so people can use h2 more frequently...

Reply:
It is specifically OpenSSL 1.0.1 (CentOS 6 for example), and macOS 10.12 and older.